pages/patcher/_how-it-works.html

```diff
@@ -0,0 +1,80 @@
+<div class="row row-page-header how-it-works-section">
+  <div class="col-xs-12">
+    <h2>How it Works</h2>
+  </div>
+  {% for category in site.data.patcher-how-it-works %} {% for item in
+  category.items %}
+  <div class="col-xs-12">
+    <div style="width: 100%">
+      <div>
+        <img
+          class="patcher-icon-user"
+          src="/assets/img/patcher/step-{{ item.step-image }}.png"
+        />
+        <h3 style="margin-top: 0px">{{ item.title }}</h3>
+      </div>
+      <img
+        class="screen-img"
+        src="/assets/img/patcher/patcher-player-cast-{{ item.step-image }}.png"
+      />
+      <div class="col-wrapper">
+        <div class="col">
+          <div class="title">
+            <img
+              class="patcher-icon"
+              src="/assets/img/patcher/with-patcher-icon@3x.png"
+            />
+            <h4 style="color: #03c5e7">With Patcher</h4>
+          </div>
+          <p class="patcher-text">{{item.with-patcher}}</p>
+        </div>
+        <div class="col">
+          <div class="title">
+            <img
+              class="patcher-icon"
+              src="/assets/img/patcher/close-icon@3x.png"
+            />
+            <h4>Without Patcher</h4>
+          </div>
+          <p>{{item.without-patcher}}</p>
+        </div>
+      </div>
+    </div>
+  </div>
+  {% endfor %} {% endfor %}
+</div>
+{% for category in site.data.patcher-how-it-works %} {% for item in
+category.items %}
+<div class="popup-screen">
+  <div class="close-popup-btn">
+    <img src="/assets/img/patcher/close-btn-icon.png" />
+  </div>
+  <img
+    class="screen"
+    src="/assets/img/patcher/patcher-player-cast-{{ item.step-image }}.png"
+  />
+</div>
+{% endfor %} {% endfor %}
+
+<script>
```

Comment:
every JS script should be started after the document so add "document ready" or "on load"